src/utils/fs.ts

```diff
@@ -278,8 +278,18 @@ export function removeDependency (options: DefinitionOptions) {
 
   // Remove the dependency from typings.
   function remove (path: string, file: string, dtsFile: string) {
-    return promiseFinally(rimraf(path), () => {
-      return transformDtsFile(dtsFile, typings => typings.filter(x => x !== file))
+    const libName: string = path.split('/').slice(-1)[0]
```

Comment:
The name is already available from the passed in options in the outer scope. It's probably easier to use that instead of trying to parse it yourself.